mathesar_ui/src/components/dropdown/Dropdown.svelte

```diff
@@ -33,8 +33,12 @@
     }
   }
 </script>
+<<<<<<< HEAD
+<Button id="tbButton" aria-controls="listbox-options" aria-labelledby="totalRows tbButton" aria-haspopup="listbox" bind:element={trigger} class={tgClasses} on:click={toggle}>
```

Comment:
I think we can remove `id` from Button. Instead of `aria-labelledby`, we can use `aria-label` and give the label directly. The label can be a prop passed on to the component.

Reply:
I implemented the changes. Not entirely sure if hardcoding the aria-label was the right approach. Let me know your thoughts. Thanks!

Reply:
The label should be a svelte property. It should not be hardcoded in the component. I'm suggesting:

```
export let ariaLabel:string = null;

<Button ... aria-label={ariaLabel}/>
```